cfpb/cfgov-refresh
# Ask landing page updates.

## Additions

- Add link between Spanish and English Ask landing pages.

## Changes

- Prevent text from wrapping under icon on Ask landing cards.

## Testing

1. Check out branch
2. Run `gulp styles`
3. Navigate to [Spanish Ask landing page](http://localhost:8000/es/obtener-respuestas/)  
    - verify link between Spanish and English versions of Ask landing 
    - verify that text does not wrap under icon on long card titles

## Screenshots

<img width="1246" alt="Screen Shot 2019-06-06 at 8 51 21 AM" src="https://user-images.githubusercontent.com/778171/59047341-84daf900-8838-11e9-82ed-d6b2e02247b1.png">
